Ksenia Myakinkaya/css/main.css

```diff
@@ -0,0 +1,87 @@
+body{
+    background-image: url(../images/main_bg.jpg);
+    background-size: cover;
+    margin: 0;
+}
+
+#black{
+    background-position: center;
+    background-size: contain;
+    background-image: url(../images/black_bg.png);
+    width: 100%;
+    height: 100vh;
+    background-repeat: no-repeat;
+    display: -webkit-flex; 
+    display: flex; 
+    flex-direction: column;
+    justify-content: center;
+}
+
+.line{
+    display: flex; 
+    flex-direction: row;
+    justify-content: center;
+}
+
+.square{
+    height: 15vw;
+    text-align: center;
+    background-size: contain;
+    border: 0.5px #bdbdbd solid;
+    flex-basis: 15vw;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', sans-serif;
+    color: #ffffff;
+    font-size: 1vw;

```

Comment:
`vw` is not so widely used for `font-size`. I suppose here you can use just `px`